control-plane/config/200-webhook/100-webhook-cluster-role.yaml

```diff
@@ -98,3 +98,13 @@ rules:
     verbs:
       - list
       - get
+      - watch
+
+  # messaging.knative.dev resources and finalizers we care about.
+  - apiGroups:
+      - messaging.knative.dev
+    resources:
+      - kafkachannels
+    verbs:
+      - get
+      - list
```

Comment:
Maybe we need `watch` here too?

Reply:
I tried locally and we don't need watch on Kafkachannels.